### PR TITLE
MTV-1554: release notes for MTV 2.7.2

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1554](https://issues.redhat.com/browse/MTV-1554)

### REVIEW

* [MTV 2.7.2 release notes](https://anarnold97.github.io/MTA-Preview/MTV/2.7.2-rn.html)

### CODE REVIEW

**Review by mnecas — APPROVED** [review]
lgtm

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 1]
```diff
@@ -7,6 +7,46 @@
...
+
+.Migration hanging due to error: virt-v2v: error: -i libvirt: expecting a libvirt guest name
+
+In earlier releases of {project-short}, when attempting to migrate a VMware VM with a non-compliant Kubernetes name, the Openshift console returned a warning that the VM would be renamed. However, after starting the Migration Plan, it hangs since the migration pod is in an `Error` state. This issue has been resolved in {project-short} 2.7.2. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1555[(MTV-1555)]
+
+.virt-v2v guest conversion the plan does not fail if the conversion pod fails
+
+In earlier releases of {project-short}, when running the `virt-v2v` guest conversion, the plan did not fail if the conversion pod failed because the error was skipped. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1569[(MTV-1569)]
```
Comment (mnecas) [inline documentation/modules/rn-27-resolved-issues.adoc:39]: Please remove this issue as we were not able to reproduce it and verify it.

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 2]
```diff
@@ -7,6 +7,46 @@
...
+
+.Warm migration is not functioning as expected
+
+In earlier releases of {project-short}, warm migration did not function as expected. When running the warm migration with VMs larger than the MaxInFlight disks, the VMs over this number did not start the migration until the cutover. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1543[(MTV-1543)]
+
+.Migration hanging due to error: virt-v2v: error: -i libvirt: expecting a libvirt guest name
+
+In earlier releases of {project-short}, when attempting to migrate a VMware VM with a non-compliant Kubernetes name, the Openshift console returned a warning that the VM would be renamed. However, after starting the Migration Plan, it hangs since the migration pod is in an `Error` state. This issue has been resolved in {project-short} 2.7.2. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1555[(MTV-1555)]
+
+.virt-v2v guest conversion the plan does not fail if the conversion pod fails
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:37]:
```suggestion
```

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 3]
```diff
@@ -7,6 +7,45 @@
...
+
+In earlier releases of {project-short}, warm migration did not function as expected. When running the warm migration with VMs larger than the MaxInFlight disks, the VMs over this number did not start the migration until the cutover. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1543[(MTV-1543)]
+
+.Migration hanging due to error: virt-v2v: error: -i libvirt: expecting a libvirt guest name
+
+In earlier releases of {project-short}, when attempting to migrate a VMware VM with a non-compliant Kubernetes name, the Openshift console returned a warning that the VM would be renamed. However, after starting the Migration Plan, it hangs since the migration pod is in an `Error` state. This issue has been resolved in {project-short} 2.7.2. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1555[(MTV-1555)]
+
+
+In earlier releases of {project-short}, when running the `virt-v2v` guest conversion, the plan did not fail if the conversion pod failed because the error was skipped. This issue has been resolved in {project-short} 2.7.2. link:https://issues.redhat.com/browse/MTV-1569[(MTV-1569)]
+
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:39]:
```suggestion
```

**Pull request merged**